pkg/plugins/platform/local.go

```diff
@@ -812,13 +812,26 @@ func (l *LocalApp) Config() error {
 func (l *LocalApp) Down(removeData bool) error {
 	l.DockerEnv()
 	settingsFilePath := l.AppConfig.SiteSettingsPath
+	_, missingConfig := CheckForConf(l.AppRoot())
 
-	err := dockerutil.ComposeCmd(l.ComposeFiles(), "down", "-v")
-	if err != nil {
-		util.Warning("Could not stop site with docker-compose. Attempting manual cleanup.")
-		err = Cleanup(l)
+	if !fileutil.FileExists(l.AppRoot()) {
+		err := Cleanup(l)
+		if err != nil {
+			util.Failed("Failed to remove %s: %s", l.GetName(), err)
+		}
+	} else if missingConfig != nil {
```

Comment:
This conditional can just be combined with the one above it, as they both do the same thing.

`if !fileutil.FileExists(l.AppRoot()) || missingConfig != nil { ... }`